content/projects/gsoc/mentors.adoc

```diff
@@ -47,18 +45,281 @@ Mentors are expected to...
 * Work in teams of 2+ mentors per 1 each student
 * Dedicate a consistent and significant amount of time, especially during the coding phase (_~5 hours_ per week in a team of two mentors)
 
-Mentorship does **NOT** require strong expertise in Jenkins plugin development.
+Mentoring does **NOT** require strong expertise in Jenkins plugin development.
 The main objective is to guide students and to get them involved into the Jenkins community.
-GSoC org admins will help to find advisors if a special expertise is needed.
+GSoC org admins will help to find advisers if a special expertise is needed.
+
+== Expectations from mentors per phase
+
+We have divided what mentors should expect in the different phases.
+Overall for mentors the time commitment is:
+
+* Organization application phase: spend the time you want.
+* Student exploration and application phase: 2 to 5 hours per week for 5.5 weeks.
+* Slot request and final selection phase: about 5 hours total over 3 weeks, but because this phase is time critical and short, we insist that you make an extra effort to attend meetings and participate actively in this process (see below for details).
+* Community bonding: 5 to 8 hours per week, this phase is critical for the success of the project, make sure you get on the ball right away.
+* Coding and evaluation periods: 5 to 8 hours per week until the end of the program. The number of hours tend to vary, more at the start, less at the end, not everyone is the same.
+
+=== Call for mentors and organization application phase
+
+*Expectation:* it is the time you spend reading about the program (2 to 3 hours) and the time you wish to spend submitting project ideas (maybe 3 to 5 hours per project idea).
+
+When you get the call for mentors announcement towards the end of December or beginning of January,
+it is time to think about joining the Jenkins GSoC program as a mentor.
+During this phase, the mentor responsibilities are:
+
+* Read this document and the related links
+* Reach out to the Jenkins GSoC community in the link:https://gitter.im/jenkins-gsoc[gitter chat]
+* Add your name as potential mentor to the project ideas you wish to mentor (in pull-requests)
+* Propose new project idea(s)
+
+You can also:
+
+* Engage with prospective students in Gitter and the mailing lists
```

Comment:
Is it worth mentioning somewhere in this doc that we expect all communications with students to be public, and that we discourage private emails?

Reply:
Is that an expectation?  I was contacted by a potential student through a gitter direct message. We used that gitter direct message to exchange ideas and test concepts. I think it was less intimidating for the student and for me because we were both very comfortable admitting when we did not understand something and when we weren't able to make progress.

Reply:
In general we discourage private communications with mentors in favor of public channels., and I am +1 for having it. But some explanation might be needed to make it less strict.

If mentors are fine with being contacted in private, it is their decision. Note that org admins will not be able to use private communications as a source of info in the case of the escalations or a dispute.

Reply:
I added it near the top of the doc.

Reply:
I actually missed what you said @MarkEWaite . I would tend to agree with you Mark. For introductions and initiating contact, direct messages make sense. The concern is sharing information that would not be available to other students, an thus put a specific student at an advantage.